sdk/Sdk/Sdk.csproj

```diff
@@ -9,8 +9,8 @@
     <RootNamespace>Microsoft.Azure.Functions.Worker.Sdk</RootNamespace>
     <CopyLocalLockFileAssemblies>true</CopyLocalLockFileAssemblies>
     <IncludeBuildOutput>false</IncludeBuildOutput>
-    <FunctionsExecutionModel>isolated</FunctionsExecutionModel>
```

Comment:
Moving this to the props file as it should be present in the application's project system